Braze Demo/AppboyManager.swift

```diff
@@ -104,14 +104,25 @@ extension AppboyManager {
     Appboy.sharedInstance()?.logCustomEvent(eventName, withProperties: properties)
   }
   
-  func setCustomAttributeWithKey(_ key: String?, andStringValue value: String?) {
+  func setCustomAttributeWithKey<T: Equatable>(_ key: String?, andValue value: T?) {
```

Comment:
Does `Equatable` simply limit the potential type of objects people pass into `value`, since all types that can be potentially entered here are `Equatable`?

Reply:
Yup, this just makes it more explicit. If somebody were to pass in a value that does not conform to `Equatable` it'll throw a compile-time error instead of just doing nothing.